File: LinkCollector-msSQL/LinkCollector-msSQL.OB/CFileManager.vb
Imports System.IO

Public Class CFileManager
    Public Sub SaveFF_net(ByVal sInput As String)

        Dim objStreamWriter As StreamWriter

        objStreamWriter = File.AppendText("fanfiction.net.txt")
        objStreamWriter.WriteLine(sInput)
        objStreamWriter.Close()
        objStreamWriter.Dispose()
        objStreamWriter = Nothing
    End Sub
    Public Sub SaveAddv(ByVal sInput As String)

        Dim objStreamWriter As StreamWriter

        objStreamWriter = File.AppendText("Addventure.txt")
        objStreamWriter.WriteLine(sInput)
        objStreamWriter.Close()
        objStreamWriter.Dispose()
        objStreamWriter = Nothing
    End Sub
    Public Sub SaveOtherBkmk(ByVal sInput As String)

        Dim objStreamWriter As StreamWriter

        objStreamWriter = File.AppendText("OtherBookmarks.txt")
        objStreamWriter.WriteLine(sInput)
        objStreamWriter.Close()
        objStreamWriter.Dispose()
        objStreamWriter = Nothing
    End Sub
    Public Sub SaveLinks(ByVal sInput As String)

        Dim objStreamWriter As StreamWriter

        objStreamWriter = File.AppendText("Links.txt")
        objStreamWriter.WriteLine(sInput)
        objStreamWriter.Close()
        objStreamWriter.Dispose()
        objStreamWriter = Nothing
    End Sub
    Public Sub SaveLink(ByVal sInputLink As String, ByVal sInputFileName As String)
        Dim objStreamWriter As StreamWriter

        objStreamWriter = File.AppendText(sInputFileName)
        objStreamWriter.WriteLine(sInputLink)
        objStreamWriter.Close()
        objStreamWriter.Dispose()
        objStreamWriter = Nothing
    End Sub
    Public Function Load(ByVal vsFileName As String) As String
        Try
            Dim oStreamReader As System.IO.StreamReader
            Dim sOutput As String

            If File.Exists(vsFileName) Then


                oStreamReader = File.OpenText(vsFileName)
                sOutput = oStreamReader.ReadToEnd
                oStreamReader.Close()
                oStreamReader = Nothing

            Else

                Dim oExp As New Exception(vsFileName & " does not exist.")
                Throw oExp


            End If


            Return sOutput

        Catch ex As Exception

            Throw ex

        End Try
    End Function
End Class

File: LinkListBuilder/FFTool.OB/CFileManager.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.IO;
namespace FFTool.OB
{
  



public class CFileManager
{

	public void SaveFF_net(string sInput)
	{
		StreamWriter objStreamWriter = null;

		objStreamWriter = File.AppendText("fanfiction.net.txt");
		objStreamWriter.WriteLine(sInput);
		objStreamWriter.Close();
		objStreamWriter.Dispose();
		objStreamWriter = null;
	}

	public void SaveAddv(string sInput)
	{
		StreamWriter objStreamWriter = null;

		objStreamWriter = File.AppendText("Addventure.txt");
		objStreamWriter.WriteLine(sInput);
		objStreamWriter.Close();
		objStreamWriter.Dispose();
		objStreamWriter = null;
	}

	public void SaveOtherBkmk(string sInput)
	{
		StreamWriter objStreamWriter = null;

		objStreamWriter = File.AppendText("OtherBookmarks.txt");
		objStreamWriter.WriteLine(sInput);
		objStreamWriter.Close();
		objStreamWriter.Dispose();
		objStreamWriter = null;
	}

	public void SaveLinks(string sInput)
	{
		StreamWriter objStreamWriter = null;

		objStreamWriter = File.AppendText("Links.txt");
		objStreamWriter.WriteLine(sInput);
		objStreamWriter.Close();
		objStreamWriter.Dispose();
		objStreamWriter = null;
	}
    public void SaveLink(string sInput)
    {
        StreamWriter objStreamWriter = null;

        objStreamWriter = File.AppendText("Links.txt");
        objStreamWriter.WriteLine(sInput);
        objStreamWriter.Close();
        objStreamWriter.Dispose();
        objStreamWriter = null;
    }
    public void SaveLink(string sInput, string sPath)
    {
        StreamWriter objStreamWriter = null;

        objStreamWriter = File.AppendText(sPath);
        objStreamWriter.WriteLine(sInput);
        objStreamWriter.Close();
        objStreamWriter.Dispose();
        objStreamWriter = null;
    }
}
}